notifications/rest/services/create-service/create-service.php

```diff
@@ -0,0 +1,16 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require './vendor/autoload.php';
+
+// Your Account Sid and Auth Token from twilio.com/user/account
+$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"; 
+$token = "{{ auth_token }}"; 
+
+// Initialize the client
+$client = new Twilio\Rest\Client($sid, $token);
+
+$service = $client->notifications->v1->services->create(array(
```

Comment:
and here. Sorry for tagging every occurrence, but it might make it easier to review as you'll see the comments disappear when the line is changed.